src/utils/externalCbnftUrl.ts

```diff
@@ -0,0 +1,12 @@
+import coke from '@/config/partners/coke'
+
+export function externalCbnftUrl({ address }: { address: string }) {
+  const allDrops = [...coke.drops]
```

Comment:
In a follow up PR, we should iterate through all partner drops and use `mintType === MintType.ThirdWeb` for whether or not we open in CBNFT.

Is there already a utility to get all partners? I didn't want to import them individually if not needed

Reply:
would want to have discussion before we use for all